src/fontra/views/editor/panel-glyph-note.js

```diff
@@ -140,9 +143,9 @@ async function saveGlyphNote(glyphName, sceneController, newNote) {
     glyph.customData["fontra.glyph.note"] = newNote;
     return oldNote
       ? newNote
-        ? "edit glyph note"
-        : "delete glyph note"
-      : "add glyph note";
+        ? translate("sidebar.glyph-note.undo.edit")
```

Comment:
Please rework to use a single `translate()` call.

Reply:
done.